hadoop-ozone/dist/src/main/smoketest/s3/objectputget.robot

```diff
@@ -26,10 +26,11 @@ Suite Setup         Setup s3 tests
 ${ENDPOINT_URL}       http://s3g:9878
 ${OZONE_TEST}         true
 ${BUCKET}             generated
+${BUCKET1}            generated
```

Comment:
Similarly, we can rename the instances of `${BUCKET}` and `${BUCKET1}` to also include the layout type.